lib/centurion/service.rb

```diff
@@ -63,10 +63,11 @@ def image=(image)
       @image = image
     end
 
-    def build_config(server_hostname, &block)
+    def build_config(server_hostname, hostname = nil)
+
       container_config = {}.tap do |c|
         c['Image'] = image
-        c['Hostname'] = yield server_hostname if block_given?
+        c['Hostname'] = hostname if hostname
```

Comment:
Is this ok? The hostname was always set before @relistan's changes, see here https://github.com/newrelic/centurion/commit/29832f27fbf7c46d4a0c60187be332e921180583#diff-51ac10da06bda45e6a706ed5d291a6dcL70